README.md

```diff
@@ -0,0 +1,705 @@
+
+Artifacts are used to transfer files between different jobs of the same workflow. The artifact is zip archive of the file without file-path included.
+
+That is also possible to download artifacts via GitHub Action's web interface.
+
+### Job: Create Pull Request with the new definition file
+
+> The job is skipped if the *'Publish to Docker Official Library'* isn't checked (`inputs.pr` set into `false`)
+
+#### Step: Checkout *official-images*, branch 'master'
+
+The [actions/checkout@v4](https://github.com/actions/checkout/) checkouts *container-images* into branch 'master'. The repository directory is located at `/home/runner/work/container-images/official-images`.
+
+That's your fork of [docker-library/official-images](https://github.com/docker-library/official-images) repository.
+
+#### Step: Sync *official-images* with upstream
+
+The step is written in bash. It adds the [official-images upstream](https://github.com/docker-library/official-images) repository, set local 'master' branch to track corresponded from the upstream, and tries to rebase with the 'upstream/master'.
+
+#### Step: Download all definitions
+
+Uses [actions/download-artifact@v4](https://github.com/actions/download-artifact) to download multiple (`merge-multiple: true`) artifacts with generated definitions. The files are saved into the `official-images/library/` directory.
+
+#### Step: Create head of *official-images/library/almalinux*
+
+Creates heading for the Docker *Definition File*.
+
+#### Step: Merge definitions into *official-images/library/almalinux*
+
+The step is written in bash. It appends the `official-images/library/almalinux` *Definition File* with downloaded on the previous step definitions `official-images/library/almalinux.version_major.image_types`.
+
+#### Step: Prepare date stamp
+
+Generates `date_stamp` in format *YYYYMMDD*. It is used in commit message and pull request title.
+
+#### Step: Prepare time stamp
+
+Generates `time_stamp` in format *HH:MM:SS*. It is used in commit message and pull request title.
+
+#### Step: Commit and push *official-images/library/almalinux*
+
+Uses [EndBug/add-and-commit@v9](https://github.com/marketplace/actions/add-commit) to commit and push the generated *Definition File*.
+
+The commit message is:
+```yaml
+Almalinux auto-update - ${{ env.date_stamp }}.
+```
+
+#### Step: Create Pull Request for *official-images/library/almalinux*
+
+The step is written in bash. It uses Github CLI to create Pull Request for the `official-images/library/almalinux` *Definition File* from your fork and to [docker-library/official-images](https://github.com/docker-library/official-images) repository.
+
+The Pull Request will be draft if the `draft` input is checked. When ready, [mark the request as ready for review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request#marking-a-pull-request-as-ready-for-review).
```

Comment:
The Pull Request will be drafted if the `draft` input is checked. When ready, [mark the request as ready for review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request#marking-a-pull-request-as-ready-for-review).

Reply:
Done.